### PR TITLE
Added info on syntax highlighting #133

### PR DESCRIPTION
**Pull request by dariavladykina**
Updated the 'Screens' subsection with the info on automatic syntax highlighting #133.

### CODE REVIEW

File: xml/docu_styleguide.structure.xml [hunk 1]
```diff
@@ -647,6 +647,24 @@ data  home  lost+found  opt     run   srv      tmp&lt;/screen&gt;</screen>
       </listitem>
      </itemizedlist>
     </listitem>
+    <listitem>
+      <para>To enable automatic syntax highlighting, add a language
```
Comment (tomschr) [inline xml/docu_styleguide.structure.xml:651]: You should mark this language attribute too with the appropriate tag:

```suggestion
      <para>To enable automatic syntax highlighting, add a <tag class="attribute">language</tag> 
```

Comment (tomschr) [inline xml/docu_styleguide.structure.xml:651]: I'm not sure if this is needed (or it's clear from the context), but "language" can be misleading. Some readers might assume we talk about _natural languages_. However, in this context, it's about _programming languages_ or _formats_.

If you want it to make it clear, here is my idea:

```suggestion
      <para>To enable automatic syntax highlighting for programming languages or formats, add a language 
```

Maybe you find a better expression.

**Review by tomschr — CHANGES_REQUESTED** [review]
Looks good Daria. I have only two minor things (one is optional).

**Review by janajaeger — COMMENTED** [review]
What Toms said. Looks great otherwise.

File: xml/docu_styleguide.structure.xml [hunk 2]
```diff
@@ -647,6 +647,24 @@ data  home  lost+found  opt     run   srv      tmp&lt;/screen&gt;</screen>
       </listitem>
      </itemizedlist>
     </listitem>
+    <listitem>
+      <para>To enable automatic syntax highlighting for programming languages or formats, add a language
```
Comment (dariavladykina) [inline xml/docu_styleguide.structure.xml:651]:
```suggestion
      <para>To enable automatic syntax highlighting for programming languages or
      formats, add a <tag class="attribute">language</tag> 
```

**Review by dariavladykina — COMMENTED** [review]
Applied review suggestions .

File: xml/docu_styleguide.structure.xml (hunk 2 above)
Comment (tomschr) [reply xml/docu_styleguide.structure.xml:651]: @dariavladykina do you add the `<tag>` element too? That's actually also important.

Comment (dariavladykina) [reply xml/docu_styleguide.structure.xml:651]: Thanks for your review @tomschr ! Could you please help me understand where to add the `<tag>` element?

Comment (tomschr) [reply xml/docu_styleguide.structure.xml:651]: Yes, see above. It's the "_...for programming languages or formats, add a **language** attribute_". The bold part needs the respective tag (also see above).

**Review by dariavladykina — COMMENTED** [review]
Added the necessary changes

**Pull request merged**